src/components/adminRegistrationApplications/pagination.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
```

Comment:
Bu dosyayı diğer listeleme sayfalarında da kullanabiliriz. Bunun için adminRegistrationApplications dosyası altından çıkarılmalıdır. ui  dosyasına alınabilir.